### ISSUE TITLE
[CF 1507] Unknown event "UNIT_VEHICLE" 

### ISSUE THREAD

**Issue by tmw-issue-import** [ascott18/TellMeWhen#1527]
<p>I have all of my TMW groups only display if it is false that I am controlling a vehicle or have my action bars overridden so that it doesn't show during those events, so I have Group Conditions to check that.&nbsp; In 8.0, it seems the variable for controlling a vehicle has changed because trying to enter/exit TMW now gives this and virtually identical LUA errors in addition to causing WoW reporting to be unable to open TellMeWhen_Options yet I still can although the TMW group fails to act properly upon exiting TMW:</p>
<div>

```
Message: ...llMeWhen\Components\Core\Conditions\UpdateEngine.lua:54: Attempt to register unknown event "UNIT_VEHICLE"
Time: Wed Jul 18 01:44:28 2018
Count: 1
Stack: ...llMeWhen\Components\Core\Conditions\UpdateEngine.lua:54: Attempt to register unknown event "UNIT_VEHICLE"
[C]: in function `RegisterEvent'
...llMeWhen\Components\Core\Conditions\UpdateEngine.lua:54: in function `UpdateTable_OnUsed'
...MeWhen\Components\Core\Common\UpdateTableManager.lua:55: in function `UpdateTable_Register'
...llMeWhen\Components\Core\Conditions\UpdateEngine.lua:100: in function `RegisterObjForEvent'
...llMeWhen\Components\Core\Conditions\UpdateEngine.lua:79: in function `RegisterObjForOnEvent'
...llMeWhen\Components\Core\Conditions\UpdateEngine.lua:187: in function `RegisterObject'
...eWhen\Components\Core\Conditions\ConditionObject.lua:232: in function `RegisterForUpdating'
...eWhen\Components\Core\Conditions\ConditionObject.lua:316: in function `DeclareExternalUpdater'
...nterface\AddOns\TellMeWhen\Components\Core\Group.lua:404: in function `Setup_Conditions'
...nterface\AddOns\TellMeWhen\Components\Core\Group.lua:476: in function &lt;...nterface\AddOns\TellMeWhen\Components\Core\Group.lua:424&gt;
(tail call): ?
[C]: ?
[string "safecall Dispatcher[1]"]:9: in function &lt;[string "safecall Dispatcher[1]"]:5&gt;
(tail call): ?
Interface\AddOns\TellMeWhen\TellMeWhen.lua:2658: in function `UpdateNormally'
Interface\AddOns\TellMeWhen\TellMeWhen.lua:2790: in function `Update'
Interface\AddOns\TellMeWhen\TellMeWhen.lua:1156: in function `?'
...ars\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:145: in function &lt;...ars\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:145&gt;
[string "safecall Dispatcher[1]"]:4: in function &lt;[string "safecall Dispatcher[1]"]:4&gt;
[C]: ?
[string "safecall Dispatcher[1]"]:13: in function `?'
...ars\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:90: in function `Fire'
...ns\GalvinUnitBars\Libs\AceEvent-3.0\AceEvent-3.0.lua:120: in function &lt;...ns\GalvinUnitBars\Libs\AceEvent-3.0\AceEvent-3.0.lua:119&gt;

Locals: &lt;none&gt;
```
</div>
<p>Predictably, the errors stop appearing when I remove all instances of Unit Controls Vehicle in the currently used Groups.</p>
<p>Edit: The LUA error about being unable to open TellMeWhen_Options is apparently a separate issue as I got that from clicking on the Open Options button in the Addons tab in the WoW Interface window but only once for some reason.&nbsp; It still opened the first time, so the game only thinks it was unable to open it for some reason, I guess.</p> 
<br>

> Posted by CurseForge user <a href="https://wow.curseforge.com/members/Kosouda">Kosouda</a> | Imported from CurseForge issue <a href="https://wow.curseforge.com/projects/tellmewhen/issues/1507">#1507</a> | <a href="https://github.com/tmw-issue-import/dump/blob/master/html/1507.html">Raw</a>
